### ISSUE TITLE
[Bug] Latest shell-pop changes throwing an error after start up when trying to run <f9>

### ISSUE THREAD

**Issue by CsBigDataHub** [seagle0128/.emacs.d#348]
### Thank you for the bug report

- [X] I am using the latest version of `Centaur Emacs`.
- [X] I checked [FAQ](https://github.com/seagle0128/.emacs.d#faq).

### Bug description

Restart emacs and run `<f9>`.

You will see - ` Symbol’s function definition is void: shell-pop--shell `

Essentially `shell-pop--shell` function is not available in new session. 

Everything works fine after doing `C-x C-e` on `shell-pop--shell` function.

### Steps to reproduce

Restart emacs and run `<f9>`.

### Expected behavior

`<f9>` should run shell-pop.

### OS

MacOS

### Emacs Version

gccemacs-28

### Error callstack

```emacs-lisp
Debugger entered--Lisp error: (void-function shell-pop--shell)
  (shell-pop--shell)
  (get-buffer-window (shell-pop--shell))
  (setq shell-pop--window (get-buffer-window (shell-pop--shell)))
  (if (window-live-p shell-pop--window) (progn (delete-window shell-pop--window) (setq shell-pop--window nil)) (setq shell-pop--window (get-buffer-window (shell-pop--shell))))
  (if (and (frame-live-p shell-pop--frame) (frame-visible-p shell-pop--frame)) nil (if (window-live-p shell-pop--window) (progn (delete-window shell-pop--window) (setq shell-pop--window nil)) (setq shell-pop--window (get-buffer-window (shell-pop--shell)))))
  shell-pop-toggle()
  funcall-interactively(shell-pop-toggle)
  #<subr command-execute>(shell-pop-toggle nil nil nil)
  ad-Advice-command-execute(#<subr command-execute> shell-pop-toggle)
  apply(ad-Advice-command-execute #<subr command-execute> shell-pop-toggle)
  command-execute(shell-pop-toggle)
```

**Comment by seagle0128** [comment]
Thanks for your reporting!

Are you using the latest version? `shell-pop--shell` is defined in `init-ui.el`, and will be loaded at startup anyway.  It's wired!

**Comment by CsBigDataHub** [comment]
> shell-pop--shell is defined in init-ui.el

Do you mean it is defined in `init-shell.el` ? I see that, but until I do `C-x C-e` on the function, emacs does not seem to find it.

> Are you using the latest version?

yes I pulled the changes from yesterday.

**Comment by seagle0128** [comment]
Please uninstall `shell-pop` package and retry. It's better clean eln files in `~/.emacs.d/eln-cache/`.

**Comment by CsBigDataHub** [comment]
> Please uninstall `shell-pop` package and retry. It's better clean eln files in `~/.emacs.d/eln-cache/`.

Ok just did that, yet the issue persists.

**Comment by CsBigDataHub** [comment]
I just copied `shell-pop--shell` function from `init-shell.el` to `custom.el` and performed a restart. It started working.

Not sure why emacs cannot read the function in `init-shell.el`. weird.

**Comment by seagle0128** [comment]
Not sure if you are using gccemacs 28.1. I am using it on macOS, and don't have this issue.

**Comment by CsBigDataHub** [comment]
Yes I am using gccemacs 28.1 on macOS. I also have a Ubuntu Linux machine where I run gccemacs 28.1 as well. Until I copy `shell-pop--shell` to `custom.el` emacs cannot find the function. 

I have exact same code you have, yet this weird issue.
https://github.com/CsBigDataHub/.emacs.d-centaur/blob/6d9dc84f6a7f44ccdf41fdb51752556aae65add8/lisp/init-shell.el#L123

**Comment by seagle0128** [comment]
Apologize for not realizing it's an issue of minibuffer style. It was fixed in cab662ce8d0a0dfb10d383f55824a3f3d83fa448.

**Comment by CsBigDataHub** [comment]
> Apologize for not realizing it's an issue of minibuffer style. It was fixed in [cab662c](https://github.com/seagle0128/.emacs.d/commit/cab662ce8d0a0dfb10d383f55824a3f3d83fa448).

No worries. All good. Thanks for fixing.
